### PR TITLE
[improvement](show) Support that user can use show data skew statement instead of admin

### LINKED ISSUE #7913: [Enhancement] Support that user can use show data skew statement instead of admin
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-doris/issues?q=is%3Aissue) and found no similar issues.

### Description

now only admin can execute show data skew statement which is inconvenient for general user

### Solution

Support that user can use show data skew statement instead of admin

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by caiconghui**
# Proposed changes

Issue Number: close #7913 

## Problem Summary:

This PR mainly do two things:
1. Support that user can use show data skew statement instead of admin
2. Fix fe ut failed caused by pr #7876
3. Fix fe ut failed caused by pr #7391

## Checklist(Required)

1. Does it affect the original behavior: (Yes)
2. Has unit tests been added: (No)
3. Has document been added or modified: (Yes)
4. Does it need to update dependencies: (No)
5. Are there any changes that cannot be rolled back: (No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by caiconghui** [comment]
ut failed cc @EmmyMiao87   show data skew cc @morningman

**Comment by EmmyMiao87** [comment]
The document could not be built correctly

File: docs/.vuepress/sidebar/en.js [hunk 1]
```diff
@@ -616,6 +615,7 @@ module.exports = [
               "SHOW CREATE FUNCTION",
               "SHOW CREATE ROUTINE LOAD",
               "SHOW DATA",
+              "SHOW DATA SKEW",
```
Comment (EmmyMiao87) [inline docs/.vuepress/sidebar/en.js:618]: This name must be same as the name of document ~

Comment (caiconghui) [reply docs/.vuepress/sidebar/en.js:618]: done

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**